homework/index.js

```diff
@@ -16,31 +30,97 @@
     xhr.send();
   }
 
-  function createAndAppend(name, parent, options = {}) {
-    const elem = document.createElement(name);
-    parent.appendChild(elem);
-    Object.keys(options).forEach(key => {
-      const value = options[key];
-      if (key === 'text') {
-        elem.textContent = value;
-      } else {
-        elem.setAttribute(key, value);
-      }
+  function renderRepositories(repo, ulRepoInfo) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repo}`;
+    fetchJSON(url, (err, repoInfo) => {
+      ulRepoInfo.innerHTML = '';
+      const li = createAndAppend('li', ulRepoInfo, { id: 'li1' });
+      const table = createAndAppend('table', li);
+      const tbody = createAndAppend('tbody', table);
+
+      const tr1 = createAndAppend('tr', tbody);
+      createAndAppend('td', tr1, { text: 'Repository:', class: 'label' });
+      createAndAppend('td', tr1);
+      createAndAppend('a', tr1.lastChild, {
+        href: repoInfo.html_url,
+        target: '_blank',
+        text: repoInfo.name,
+      });
+
+      const tr2 = createAndAppend('tr', tbody);
+      createAndAppend('td', tr2, { text: 'Description:', class: 'label' });
+      createAndAppend('td', tr2, { text: `${repoInfo.description}` });
```

Comment:
There is not need to use a template literal for something that is already a string (or can be automatically converted to string):

```js
createAndAppend('td', tr2, { text: repoInfo.description });
```

See line 47 where you already used did this.

Check other places in your code where you have unneeded template literals.